util/util.py

```diff
@@ -185,6 +186,82 @@ def tensor2im(input_image, imtype=np.uint8):
     return image_numpy.astype(imtype)
 
 
+def tensor2im_re(input_image, imtype=np.uint8):
```

Comment:
This function already exists, we don't really want a new one as we´d need to maintain both, let's integrate the detach as an option to the existing `tensor2im` function.